### PR TITLE
Fix return type missing for Image.load() and Image.createFrom()

### LINKED ISSUE #515: Return type missing from `.load()` and `.createFrom()`
https://github.com/image-js/image-js/blob/1efc4d0470b18fd141948c2b2c005b644701e8bf/index.d.ts#L28

It looks like the returntype is `Image` in both cases, but these are missing so they come back as `any`.

### PR DESCRIPTION
**Pull request by adam-coster**
Closes #515

### CODE REVIEW

**Comment by stropitek** [comment]
Thanks!

File: index.d.ts [hunk 1]
```diff
@@ -24,8 +24,8 @@ export declare class Image {
 
     static isImage(object: any): boolean;
     static fromCanvas(canvas: HTMLCanvasElement): Image;
-    static createFrom(other: Image, options: ImageConstructorOptions);
-    static load(image: (string|ArrayBuffer|Uint8Array), options?: RequestInit)
+    static createFrom(other: Image, options: ImageConstructorOptions): Image;
+    static load(image: (string|ArrayBuffer|Uint8Array), options?: RequestInit): Image;
```
Comment (targos) [inline index.d.ts:28]: This method returns a Promise<Image> IIRC

Comment (stropitek) [reply index.d.ts:28]: Yes you're right, I overlooked that

Comment (stropitek) [reply index.d.ts:28]: fixed in 10dae9be2b5ca9c71063a7b1e7db122e2dbd787a

**Pull request merged**